patch/gfx9/llpcNggPrimShader.cpp

```diff
@@ -5291,6 +6360,129 @@ Value* NggPrimShader::DoSubgroupBallot(
     return pBallot;
 }
 
+// =====================================================================================================================
+// Output a subgroup inclusive-add (IAdd).
+Value* NggPrimShader::DoSubgroupInclusiveAdd(
```

Comment:
can you call exist builder functions?

Reply:
No. LLPC builder will not interact with any pass of patch phase.